lib/ansible/plugins/inventory/__init__.py

```diff
@@ -159,6 +159,23 @@ def __init__(self):
         self._options = {}
         self.inventory = None
         self.display = display
+        self.templar = None
+
+    def get_option(self, option):
+
+        value = super(BaseInventoryPlugin, self).get_option(option)
+
+        if self.templar.is_template(value):
+            value = self.templar.template(value)
+
+        if getattr(value, '__ENCRYPTED__', False):
```

Comment:
We have this in config manager now for string types https://github.com/ansible/ansible/blob/stable-2.10/lib/ansible/config/manager.py#L147-L155

I think this just needs a way to enable/disable templating? Maybe at the plugin level and option level, so a plugin needs to opt into it but can still disable it for static options.

Reply:
@s-hertel I am curious why it should be disabled for lookup plugins? That's where I want to use it the most, so.. it's quite relevant to me 😅